### PR TITLE
Add rmilter and deps

### PR DESCRIPTION
**Pull request by jprjr**

### CODE REVIEW

File: srcpkgs/rmilter/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+rmilter_homedir="/var/lib/rmilter"
+
+# TODO fix memcached functionality when
+# cross-compiling
+case "${XBPS_TARGET_MACHINE}" in
+	*-musl)
+		configure_args+=" -DENABLE_MEMCACHED=off"
+		;;
+	*)
+		makedepends+=" libmemcached-devel"
```
Comment (xtraeme) [inline srcpkgs/rmilter/template]: Please make this a build option and disabled on musl.

File: srcpkgs/opendkim/template [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+hostmakedepends="pkg-config"
+short_desc="DKIM library"
+maintainer="John Regan <john@jrjrtech.com>"
+license="BSD, Sendmail"
+homepage="http://www.opendkim.org"
+distfiles="$SOURCEFORGE_SITE/project/$pkgname/$pkgname-$version.tar.gz"
+checksum=43a0ba57bf942095fe159d0748d8933c6b1dd1117caf0273fa9a0003215e681b
+
+if [ -n "$CROSS_BUILD" ]; then
+	configure_args+=" --with-milter=${XBPS_CROSS_BASE}"
```
Comment (xtraeme) [inline srcpkgs/opendkim/template]: Get rid of this conditional and set `--with-milter=${XBPS_CROSS_BASE}/usr` instead.

**Comment by jprjr** [comment]
Fixed+rebased. I think I mis-understood the rmilter and memcached error, I was able to build rmilter in my x86_64-musl vm with memcached enabled, without any issues.

**Comment by Duncaen** [comment]
* 94d2d9b New package: rmilter-1.8.5
* d91e417 New package: opendkim-2.10.3
* 651988f New package: libmilter-1.0.2

**Pull request closed without merging**